### ISSUE TITLE
Descriptions do not fully reset after pressing Reset All button

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#322]
There are two descriptions that are not fully resetting after the Reset All button is pressed.
1. The sentence that describes the number of charges on John's body, "John has 0 charges on his body," should disappear after the Reset All button is pressed.
2. The position description for the Hand position slider, should reset to not include any directional information. 

On page load both descriptions are correct, but after some interaction and a reset, sentence 1 is remaining (John has 0 charges on his body), and the hand position if still at -4 contains directional information instead of state information, alone (Position -4: Away from doorknob).

My understanding is that after a reset, the sim should return to its original state, the same as page load. This would mean that sentence 1 should not be present, and that the position description for hand position -4 should be, "Position -4: Hand pointing at upper door, close to doorknob."

**Comment by terracoda** [comment]
@mbarlow12 or @jessegreenberg  could you two assign the appropriate developer and make sure these descriptions are fixed on the next release.

Sentence 1 is really not a huge problem, but the incorrect description for hand position -4, though not that serious, is a little awkward.

Please assign the priority level, too.

**Comment by jessegreenberg** [comment]
Verified, thanks for pointing out @terracoda.

**Comment by jessegreenberg** [comment]
I looked into this. In addition, the value text isn't updated on blur like it should be. So whenever focus leaves the arm and comes back, it still says "Towards doorknob".

**Comment by jessegreenberg** [comment]
I think we need the ability to set the aria-valuetext independently from the changing value.

**Comment by jessegreenberg** [comment]
Above commit should fix the issue with the aria-valuetext.

**Comment by terracoda** [comment]
Thanks @jessegreenberg! The descriptions are resetting now.
I am assuming this issue is ok to close and that the changes will be pushed to production on the next update release for this sim.
